src/redisearch_rs/trie_rs/src/trie.rs

```diff
@@ -1,13 +1,8 @@
-/*
- * Copyright (c) 2006-Present, Redis Ltd.
- * All rights reserved.
- *
- * Licensed under your choice of the Redis Source Available License 2.0
- * (RSALv2); or (b) the Server Side Public License v1 (SSPLv1); or (c) the
- * GNU Affero General Public License v3 (AGPLv3).
-*/
-
-use crate::{node::Node, utils::strip_prefix};
+use crate::{
+    iter::{Iter, LendingIter, Values, filter::VisitAll},
+    node::Node,
+    utils::strip_prefix,
+};
```

Comment:
In some files there's still a copyright and license info (lib.rs, trie_ops.rs) and in some it is removed?

Reply:
Fixed in 2afdd300a